misk/web/@misk/dev/webpack.config.tab.js

```diff
@@ -75,36 +106,41 @@ module.exports = (env, argv, otherConfigFields = {}) => {
         {
           test: /\.(tsx|ts)$/,
           exclude: /node_modules/,
-          loaders: 'awesome-typescript-loader'
+          loaders: "awesome-typescript-loader"
         },
         {
-          enforce: 'pre',
+          enforce: "pre",
           test: /\.js$/,
-          loader: 'source-map-loader'
+          loader: "source-map-loader"
         },
         {
           test: /\.scss$/,
-          loader: 'style-loader!css-loader!sass-loader'
+          loader: "style-loader!css-loader!sass-loader"
         },
         {
           test: /\.(jpe?g|png|gif|svg)$/i,
-          loader: 'url-loader',
+          loader: "url-loader",
           options: {
             limit: 10000
           }
         }
       ]
     },
     resolve: {
-      extensions: ['.js', '.jsx', '.ts', '.tsx', '.json']
+      extensions: [".js", ".jsx", ".ts", ".tsx", ".json"]
```

Comment:
Not a deal breaker, but I'm curious to know if there a reason we chose to go for double quotes in JS over single quotes?

Reply:
From my research it seems to be the recommended for use with Typescript.

https://github.com/Microsoft/TypeScript/issues/5811